spec/features/admin/items/admin_can_edit_an_item_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+describe "when admin visits the items page" do
+  it "admin can create a new item" do
+    adminda = Fabricatior(:user)
+    adminda.admin!
+    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(adminda)
+    visit new_item_path
+
+    expect(page).to have_content("Create A New Item")
+
+    category = Category.create(name: "Postions")
+    item_one = Item.new(title: "Love Posions",
+                    description: "For all your foolish needs",
+                    price: 12.78,
+                    image_tage: "http://harrypotter.com/love_postion"  #you're going to want to change this
+    item_two = Item.new(title: "Hair Growing Posions",
+                    description: "Grow your hair, grow your teachers hair, get it here",
+                    price: 120.78,
+                    image_tage: "http://harrypotter.com/hair_growing_postion"  #you're going to want to change this
```

Comment:
could use Fabricate(:item) here as a refactor. curious if you ran into any problems using it?